browser/ui/android/strings/android_brave_strings.grd

```diff
@@ -178,9 +178,6 @@ This file contains all "about" strings.  It is set to NOT be translated, in tran
       <message name="IDS_ACCESSIBILITY_TOOLBAR_BTN_BRAVE_REWARDS" desc="Content description for the button that is enable, disable Brave Rewards.">
         Manage Brave Rewards
       </message>
-      <message name="IDS_PREFS_SECTION_SEARCH_ENGINE" desc="Title for search engine section that can select DSE of normal and private tab">
```

Comment:
Are all these strings not used?

Reply:
yes, not used anymore (some were leftovers from a previous design too). i went through the warnings to identify all of them